python/lsst/dax/imgserv/fitsMetadataSchema.py

```diff
@@ -0,0 +1,59 @@
+
+
```

Comment:
Thanks @timj, it looks like we need a ticket to purge all copyrights from every file. I can review that :smile: